nni/experiment/management.py

```diff
@@ -2,12 +2,22 @@
 # Licensed under the MIT license.
 
 from pathlib import Path
+import logging
 import random
 import string
 
+_logger = logging.getLogger(__name__)
+
 
 def generate_experiment_id() -> str:
-    return ''.join(random.sample(string.ascii_lowercase + string.digits, 8))
```

Comment:
I don't think it worth introducing a dependency.
Replacing `random.sample()` with `Random().sample()` should be enough.